cohort-engine/pom.xml

```diff
@@ -108,6 +108,12 @@
 			<version>${project.version}</version>
 		</dependency>
 
+		<dependency>
```

Comment:
In the spirit of more descriptive PRs, would be interested in an overview of what this library is providing for us (I did google it but feel like having it here would be beneficial too)